### PR TITLE
feat: Add contents to the Document vertex

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

File: com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/LsifUtils.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+public class LsifUtils {
+
+	private LsifUtils() {
+	}
+
+	/**
+	 * Normalize the URI to the same format as the client.
+	 */
+	public final static String normalizeUri(String uri) {
```
Comment (Eskibear) [inline com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/LsifUtils.java]: cannot easily understand what it's doing from `normalizeUri`:

0-7: `file:///`
8: Upper Letter?   -> the purpose is to change this byte to lower?
9: `:`

File: com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/JdtlsUtils.java [hunk 2]
```diff
@@ -161,19 +166,6 @@ public static ISourceRange getNameRange(IJavaElement element) throws JavaModelEx
 		return nameRange;
 	}
 
-	/**
```
Comment (Eskibear) [inline com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/JdtlsUtils.java]: I know it's originally from here, and what's the purpose of moving it to lsifUtil?

Comment (jdneo) [reply com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/JdtlsUtils.java]: `JdtlsUtils` is to out the utilities referenced in `JDTUtils`

`lsifUtil` is to put general utilities used by the indexer

File: com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/LsifUtils.java (hunk 1 above)
Comment (jdneo) [reply com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/LsifUtils.java]: Basically it's dealing with the case of the driver on Windows. e.g.: Change `C:\\` to `c:\\`

**Comment by jdneo** [comment]
> I don't have much context info. So the purpose of this PR is to set a base64 encoded content to the instance?

Exaclty

**Pull request merged**